api/src/controllers/user/getUser.ts

```diff
@@ -0,0 +1,46 @@
+import { Request, Response } from 'express';
+
+import NotAuthorizedException from '../../exceptions/NotAuthorizedException';
+import * as userRepository from '../../repositories/userRepository';
+import { decodeQueryToUser } from '../../util/helper';
+import controller from '../controllerUtil';
+import Validator, { beProperlyUriEncoded } from '../validation';
+
+type ReqParams = {
+    user: string;
+};
+
+class ReqParamsValidator extends Validator<ReqParams> {
+    constructor() {
+        super('path parameters');
+
+        this.ruleFor('user').mustAsync(beProperlyUriEncoded);
+    }
+}
+
+type ResBody = {
+    exists: boolean;
+};
+
+/**
+ * Get my resume reviews.
+ * @param req HTTP request.
+ * @param res HTTP response.
+ * @returns Your resume reviews.
+ */
+const getUser = controller(async (req: Request<ReqParams, ResBody, unknown, unknown>, res: Response<ResBody>): Promise<void> => {
```

Comment:
You can just stop adding generics once you don't want to change them anymore. In other words `Request<ReqParams, ResBody>`. You would need those unknowns if you were doing something like `Request<ReqParams, ResBody, unknown, unknown, AnotherType>`

Reply:
Ah gotcha, thanks for letting me know. Will update 👍